autonomx/main.qml

```diff
@@ -120,6 +124,38 @@ ApplicationWindow {
         }
     }
 
+    // serialization
+    Shortcut {
+        id: saveSC
+        sequence: "Ctrl+S"
```

Comment:
The shortcuts could be made more portable (especially on macOS) by using StandardKey like in https://doc.qt.io/qt-5/qml-qtquick-shortcut.html#sequences-prop